platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/migration/SystemConfigurationMigrationTest.java

```diff
@@ -516,6 +557,14 @@ private Path createTempDdf() throws IOException {
         Files.createFile(rootTempDir.resolve(INVALID_KEYSTORE_REL_PATH));
         Files.createFile(rootTempDir.resolve(INVALID_TRUSTSTORE_REL_PATH));
         Files.createFile(rootTempDir.resolve(INVALID_CRL_REL_PATH));
+
+        existingFilePath = Paths.get(ddfHome.resolve("orig.config")
```

Comment:
Any reason we store `existingFilePath` as a private field and we don't just do like the other calls above, i.e., `Files.createFile(ddfHome.resolve("orig.config"))`?